modules/link-prediction/pages/index.adoc

```diff
@@ -0,0 +1,19 @@
+= Topological Link Prediction Algorithms
+:description: Overview of link prediction algorithms.
+
+Link prediction algorithms determine the closeness score of a pair of vertices. This score can then be used to make predictions about the relationships between the two vertices in the pair.
```

Comment:
We should start the sentence with "Topological". That is because our algorithms are limited to that subcategory of link prediction algorithm. In general, link prediction is just that: what is the probability that there is a link here? Basing that prediction on the similarity of the two vertices is only one way to make that prediction.